### ISSUE TITLE
Project home page

### ISSUE THREAD

**Issue by kevinwinahradsky** [480Oswego2013/CSC-HCI-480-2013-repo#54]
This is what we have now

http://cs.oswego.edu/~jwhite8/csc480/

Any last minute thoughts on how to make that a little nicer? I personally wouldn't mind just seeing the maven generated site thrown up there.

**Comment by patricktoohey** [comment]
At the very least, we need a link to the github site from a class related
oswego.edu address.

In general, I agree that the existing implementation is insufficient.
However, unless we are going to host H2O somewhere as well, I don't think
included the actual application is worthwhile.

**Comment by 480Oswego2013** [comment]
ill work on this web page some more and see if i can't get it looking a little better

**Comment by patricktoohey** [comment]
I think you'd want to include the basics from the google site; {school
logo, class name, semester, project name, github link}.

**Comment by 480Oswego2013** [comment]
thanks! check again around 3 and see what you think.

**Comment by adarwin** [comment]
I think the maven generated site is probably a good fallback, if we decide we don't like what we have at 3:00
That said, I tried running `mvn site` but it never produced the full website. index.html was missing, for starters.

**Comment by 480Oswego2013** [comment]
how is the maven site created? is it a terminal command or a netbeans thing?

**Comment by 480Oswego2013** [comment]
sounds like a plan

**Comment by adarwin** [comment]
Terminal, I believe

**Comment by patricktoohey** [comment]
navigate to the CSC-HCI-480-2013-repo folder in the git checkout. execute
'mvn site' from the console.

**Comment by 480Oswego2013** [comment]
is that inside netbeans or just on the git site? I’m on the site but it did not pop out to me.

**Comment by patricktoohey** [comment]
Maven is included with netbeans; on windows, for example, located in
"NetBeans 7.3\java\maven". Can also be downloaded independently. I just
built the site locally. If we use it, I would say, update the 'About' page
to include general summary and github reference and 'Project' summary page
to point to github reference instead of google site (which is locked down).

**Comment by 480Oswego2013** [comment]
ill give it a shot my webpage skills are not top notch. it would be acceptable at best
im in the maven dir but still just a little confused on how this site is built

**Comment by patricktoohey** [comment]
just worry about adding content within the existing structure. Can pull
content from the google site if it helps.

**Comment by 480Oswego2013** [comment]
how can i update to maven 3 i can almost build the site, then adding content should not be complicated.

**Comment by patricktoohey** [comment]
http://maven.apache.org/download.cgi

**Comment by 480Oswego2013** [comment]
i downloaded it but cant seem to install it. i followed the direction in the readme maybe to literaly

**Comment by patricktoohey** [comment]
I've placed my output here...
https://dl.dropboxusercontent.com/u/34547970/site.zip

Grab it and take a look. Edit where relevant and let us know if you get
stuck.

**Comment by 480Oswego2013** [comment]
thank you!

**Comment by 480Oswego2013** [comment]
sorry for the wait here is then new model
http://moxie.oswego.edu/~jwhite8/hViewO/ 

the only thing i am having trouble with is in the Project report section Javadoc and TestJavaDoc but ill make those and put them in manually in a sec

**Comment by patricktoohey** [comment]
Doesn't look quite right. Not sure why

**Comment by 480Oswego2013** [comment]
it is very blank, i could put the school logo on the front page to make it look a little better.

**Comment by adarwin** [comment]
The original seems to look fine on moxie. I'm guessing you broke something when you modified the site.

http://moxie.oswego.edu/~darwin

**Comment by kevinwinahradsky** [comment]
Have you guys updated your git workspace? I tweaked the maven stuff this morning but I do not see the updates in the generated site.

**Comment by kevinwinahradsky** [comment]
http://moxie.oswego.edu/~winahrad/hViewO/

to download it http://moxie.oswego.edu/~winahrad/site.tgz

**Comment by patricktoohey** [comment]
That is what my original looked like. Not sure what was broken.

**Comment by 480Oswego2013** [comment]
that looks way better, i must have broken something in index.html without knowing
i vote this the official page since, i boke mine some how....Thanks Kevin!!!

**Comment by nonelement** [comment]
@480Oswego2013  -- If you want a hand looking over the project site, I'm not bad with web stuff. It looks like the majority of what we're dealing with here is auto-generated?

**Comment by 480Oswego2013** [comment]
yes. maven built the site, I some how killed mine, but then again i never had maven 3 so i had no real idea what it looked like. the plan might be to just use kevins if that's legit with him. it would be easier to update anyway. kevin did you chmod your files so they could be edited?

**Comment by kevinwinahradsky** [comment]
sure, if we want to change some of the text we can just update the pom file and regenerate.

**Comment by 480Oswego2013** [comment]
ha, way smarter then what i was thinking, i should have learned maven. but that were i am hung up, i cant use the mvn site command on our project because of my version of netbeans. its not EE so it only has 2.2.1 maven on it, and i suck at updating versions

**Comment by adarwin** [comment]
I definitely pulled this morning, but it must have been before Kevin made changes. My bad.

Either way, Kevin seems to be on top of this one. I say, just make a few minor tweaks as necessary and we'll call it good.

**Comment by adarwin** [comment]
Oh, wait. Not my bad. I uploaded Patrick's version. :P

**Comment by 480Oswego2013** [comment]
i'm with drew on this one. sorry to put more on you kevin. 

i Think we should take patrics earier suggestion and throw the school logo on the page along with a thing about developers, i can do this part if you want.

**Comment by patricktoohey** [comment]
Ran the maven site again; much larger now. If needed here is a link to the
contents... https://dl.dropboxusercontent.com/u/34547970/site_4PM.zip

**Comment by adarwin** [comment]
Where do we ultimately want to host the site?

**Comment by 480Oswego2013** [comment]
i'm not sure as long as its on moxie i don’t think dl really cares.

**Comment by adarwin** [comment]
If that's the case, maybe it comes down to a battle of who has the best name. ;)

**Comment by patricktoohey** [comment]
can we count on individual moxie accounts persisting infinitely? or do they go away after graduation or some other time limit expires?

**Comment by 480Oswego2013** [comment]
most times they are not deleted, though that might change in the near future. i guess this is a dl question honestly

**Comment by adarwin** [comment]
As I understand it, individual moxie accounts persist for a couple years by default, but can receive special permission to remain longer if need be.

**Comment by 480Oswego2013** [comment]
so it really does come down to the best name, if only one of us had the last name HviewO

**Comment by 480Oswego2013** [comment]
@nonelement i was wondering how i could put the suny oswego logo on the webpage to so that it would not take up the top left corner but have a h1 and image tags right next to each other.

**Comment by 480Oswego2013** [comment]
improved a little 

http://moxie.oswego.edu/~jwhite8/csc480/project-info.html

**Comment by nonelement** [comment]
@480Oswego2013 You're probably looking to apply a CSS rule that says "display: inline-block;" for all desired elements.

**Comment by 480Oswego2013** [comment]
oh really, did not know that, I barely touch css. i will look deeper into how a inline block and display work

**Comment by adarwin** [comment]
If all else fails, maybe this can be our plan B?

http://moxie.oswego.edu/~darwin/hViewO

**Comment by nonelement** [comment]
@480Oswego2013 Images have "display: block" by default, but this causes the element to take up 100% width on the page (like headers, divs, whatever), while inline flows (like text). With "display: inline-block", the element will flow with other content, but retain it's shape and not collapse (think div vs span behavior). If this helps.

**Comment by 480Oswego2013** [comment]
Grave the mail list from mine and make it plan a

**Comment by adarwin** [comment]
Done: http://moxie.oswego.edu/~darwin/hViewO/mail-lists.html

**Comment by kevinwinahradsky** [comment]
Would that be better placed under the "Project Team" link?

**Comment by adarwin** [comment]
How's this? http://moxie.oswego.edu/~darwin/hViewO/team-list.html

**Comment by kevinwinahradsky** [comment]
looks good

**Comment by adarwin** [comment]
Changed most of the links to open in new tabs.

Anyone think of anything else I should add to the site?
